UnityProject/Assets/Scripts/UI/Right click/Button.prefab

```diff
@@ -174,12 +284,58 @@ MonoBehaviour:
   m_EffectColor: {r: 0, g: 0, b: 0, a: 0.14509805}
   m_EffectDistance: {x: 0, y: 2}
   m_UseGraphicAlpha: 1
---- !u!114 &114957084443620552
+--- !u!1 &1527225835142163805
+GameObject:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  serializedVersion: 6
+  m_Component:
+  - component: {fileID: 3420991853773627107}
+  - component: {fileID: 2655520020073026784}
+  - component: {fileID: 7175192079277077561}
+  m_Layer: 5
+  m_Name: BGPanel
```

Comment:
You didn't need to add this panel , since the circle is technically the background, You modify the sprite of the circle